rfc/0029-community-amendment-1.md

```diff
@@ -0,0 +1,325 @@
+specified.
+
+### Admin Board Members
+
+Unless they're also a voting member of the Community Team, members of the Quilt Admin Board do note vote on internal
+polls by default. Instead, they should only vote in order to break ties or contribute to inconclusive polls - votes
+provided by members of the admin board are tripled, meaning that **one vote becomes three** - this is detailed in
+[RFC 6: Governmental Structure](/community/0006-governance.md), as the admin board is the only parent of the community
+team.
+
+### Trainee Community Moderators
+
+Trainee Community Moderators are not voting members of the Community Team, and should not be considered for internal 
+votes. This is to ensure that all voters have had some time to get familiar with the community and how it's run - but
+non-voting members of the Community Team are always welcome to express their opinions and provide insight if they wish 
+to.
+
+### Voting Process
+
+1. The Community Team needs to discuss the subject matter of the vote beforehand, and discern whether a vote should
+   take place to begin with - if it's decided that a vote will surely fail, then no vote is required and the motion
+   can be dismissed immediately.
+1. A poll is created in an internal channel that's designated for that purpose, either by posting a simple message or
+   via a bot designed to facilitate polling. This poll should provide as much detail as possible, either directly in
+   the poll message or by providing links and references - but it must be clear and not misleading, and the core idea
+   must be present in the message instead of in the references.
+   
+   Polls must also include a closing date of no less than one day and longer than one week, regardless of the proposal 
+   put forward.
+1. Members of the Community Team may vote on the poll using the provided reactions. All polls should include a positive
+   and negative option, as well as the option to abstain. Voting members that abstain are excluded from percentages and
+   tallies.
+1. Voting members are not required to explicitly explain their votes - major problems with the proposal should be 
+   worked out before the vote even begins, to avoid putting particular community members on the spot.
+
+### Closing Votes
+
+Voting ends when any of the following conditions have been reached:
+
+* The poll's closing date has been reached
+* The poll has failed or passed early due to at least 75% of the voting members of the Community Team voting in that
+  direction, excluding abstaining members from that count
+* The special closing conditions specified as part of the specific process that this vote is part of have been met, if
+  any
+
+### Interpreting the Vote
+
+When a poll has closed, the votes should be inspected. If the final vote has a margin of three votes or fewer, 
+non-voting members of the administrative board must be invited to vote on the poll, with their votes counting for 
```

Comment:
> non-voting members of the administrative board must be invited to vote on the poll, with their votes counting for three

If the number of non-voting Admins is even they could also tie, which doesn't seem ideal. Not sure if we want to tweak the process here somehow or leave as is.

Reply:
I'll change this to include admins that have already voted as well, that was an oversight on my part.